Dockerfile

```diff
@@ -17,9 +18,12 @@ ARG PYINSTALLER_VERSION=v5.7.0
 RUN python3 -m venv /opt/venv
 ENV PATH="/opt/venv/bin:$PATH"
 
-RUN if [ "$TARGETARCH" != "amd64" ]; then curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y; fi
+WORKDIR /app
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
 ENV PATH="/root/.cargo/bin:${PATH}"
-RUN if [ "$TARGETARCH" != "amd64" ]; then rustup default nightly-2021-03-24; fi
+RUN rustup default nightly-2021-03-24
```

Comment:
while it is not related to the PR directly do you think we can update from this old nightly?

Reply:
did you pin it for some reason?

Reply:
Wasn't me
![image](https://thumbs.gfycat.com/SpotlessAbleFireant-size_restricted.gif)

It was an external contributor that made the aarch64 docker contribution.

But it was around two years ago so it was probably the last one around that time and whatever feature was missing, it should have landed to `stable` after two years.